src/CMakeLists.txt

```diff
@@ -409,6 +409,7 @@ endif()
 
 if(WIN32)
   list(APPEND ceph_common_deps ws2_32 mswsock)
+  list(APPEND ceph_common_deps dlfcn_win32)
```

Comment:
could you revise the title of the commit message.

could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes. in this case, it could be "common:".

Reply:
Sure, I just wasn't aware that "common" is considered to be a subcomponent. I'll change the commit title right away.